app/views/accounts/index.html.haml

```diff
@@ -0,0 +1,18 @@
+-#  Copyright (c) 2008-2017, Puzzle ITC GmbH. This file is part of
```

Comment:
there's no accounts#index route. list of accounts is shown in groups#show, isn't it ?